observer/probers/prober.go

```diff
@@ -40,7 +45,13 @@ type Configurer interface {
 	// bound `Configurer` object. If the `Configurer` cannot be
 	// validated, an error appropriate for end-user consumption is
 	// returned instead.
-	MakeProber() (Prober, error)
+	MakeProber(map[string]prometheus.Collector) (Prober, error)
```

Comment:
I'd augment the doccomment here to indicate that the argument should be the same as the return value from Instrument().

Reply:
Done!